scripts/configs/pg4.yaml

```diff
@@ -0,0 +1,53 @@
+# PG4 experiments.
+---
+APPROACHES:
+  pg3:
+    NAME: "pg3"
+    FLAGS: []
+  pg4:
+    NAME: "pg4"
+    FLAGS: []
+  plan-only:
+    NAME: "oracle"
+    FLAGS: []
+ENVS:
+  cover-easy:  # downward refinable. PG3 should succeed
+    NAME: "cover"
+    FLAGS:
+      cover_initial_holding_prob: 0.0
+  cover:  # PG3 should sometimes fail, but PG4 should succeed
+    NAME: "cover"
+    FLAGS: []
+  painting-easy-no-holding:
```

Comment:
do we know that this will succeed? should we do box-only?

Reply:
PG3 fails sometimes when it is initially holding, PG4 never fails. What do you mean by box-only?

Reply:
I meant setting the goal to be box only. maybe let's add that as a separate env?